netbox_netprod_importer/__main__.py

```diff
@@ -75,17 +77,17 @@ def parse_args():
     args = arg_parser.parse_args()
 
     if hasattr(args, "func"):
+        if args.verbose:
+            logging.basicConfig(
+                level=logging.DEBUG if args.verbose == 'debug' else logging.WARNING,
```

Comment:
Simple quotes -> double quotes

Reply:
Also, what do you think about just doing `level=getattr(logging, args.verbose.upper())`?
This way, you can specify all level you want.

Reply:
I read the documentation Logging, completely changed the approach and redid :)